source/copilot.txt

```diff
@@ -0,0 +1,80 @@
+.. _vsce-copilot:
+
+==================================
+MongoDB Github Copilot Participant
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+
+|vsce-full| includes a MongoDB Github Copilot Participant to assist in
+using `Github Copilot <https://github.com/features/copilot>`__ with your
+MongoDB deployments. Through Github Copilot's chat feature, users with
+|vsce| can interact with their MongoDB clusters and generate code with
+MongoDB domain-specific knowledge on top of Github Copilot's LLM. 
+
+The MongoDB Github Copilot Participant includes MongoDB-specific
+commands to assist in interacting with your deployment.
+
+/query
+------
+
+The ``/query`` command assists in generating queries from a natural
+language against a connected MongoDB cluster. The MongoDB Github Copilot
+Participant provides underlying schema information of the relevant
+collections to Github Copilot to generate a response. If you do not
+specify a collection in your prompt, the chat prompts you to select a
+relevant collection. 
+
+When the LLM generates a query, you can open the query in a playground
+file or run the query directly in your collection. 
+
+Example
```

Comment:
Could we provide an example where we can show that users can iteratively build an aggregation with Copilot? Sent over a view of this example to you over slack!

Reply:
Done! Because adding this example requires more page space, I made the /query section a separate page so that we can add longer examples for each command without resulting in one super long page.